src/app/Components/ComponentsSamples/ContextMenus/ContextMenuSamples.xaml

```diff
@@ -186,6 +186,37 @@
                 </dui:ContextMenu>
             </dui:ListItem.ContextMenu>
         </dui:ListItem>
+        
+        <dui:ListItem Title="{x:Static localizedStrings:LocalizedStrings.Context_Menu_Divider}"
+                      HasBottomDivider="True">
+                <dui:ListItem.ContextMenuOptions>
+                    <dui:ContextMenuOptions Mode="Pressed" />
+                </dui:ListItem.ContextMenuOptions>
+            
+            <dui:ListItem.ContextMenu>
+                
+                <dui:ContextMenu ItemClickedCommand="{Binding ItemClickedCommand}">
+                    <dui:ContextMenuItem
+                        IsDestructive="True"
+                        Title="{dui:StringFormat {x:Static localizedStrings:LocalizedStrings.Action_Format}, Argument=1}"
+                        Icon="{dui:Icons internalmessage_fill}">
+                    </dui:ContextMenuItem>
+                    <dui:ContextMenuSeparatorItem />
```

Comment:
Bruk samme betegnelse som Divider apiet, ContextMenuDivider

Reply:
Denne bør muligens ha IsVisible property?

Reply:
Etter å ha lest litt mer på det du har gjort, har jeg innsett at dette er vel mer en "gruppe divider". Altså for å skille grupperte funksjoner sammen? Kontekst menu har jo innebygd dividers mellom hvert item, så det er ikke helt det samme som en divider dette.

Reply:
Ser også at du hadde lagt på IsVisible, my bad.

Reply:
Jeg godkjenner, navnet er greit :)